### PR TITLE
Added detailed documentation for the tools command "wget"

### LINKED ISSUE #455: Writing Documentation for "wget" command

### PR DESCRIPTION
**Pull request by edricguo**
# Advanced HW 5 submission

Closes #455  

Name: Edric Guo
Uniqname: edricguo

***What I did:
Added a paragraph description and a simple example on the front page of wget.
Added useful usage and some advanced examples of wget.
***

***What to check:
Text display is intact.
***

### CODE REVIEW

File: _commands/tools/wget.md [hunk 1]
```diff
@@ -3,22 +3,48 @@
...
+
+\-\-convert-links #convert links for offline viewing
+
+\-\-user-agent=C4CS #fake the User Agent
 
 #### Example command
 
-##### Break it down
+~~~ bash
+$ wget ‐‐output-document=filename.html example.com
```
Comment (AmritHariharan) [inline _commands/tools/wget.md]: these are two different example commands, they should have their own subheading, code block, and explanation

File: _commands/tools/wget.md [hunk 2]
```diff
@@ -3,22 +3,48 @@
...
+
 ~~~ bash
-*short* example of command usage and output
+$ wget http://example.com/file.iso
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
+\-\-mirror #turn on mirroring
```
Comment (AmritHariharan) [inline _commands/tools/wget.md]: this isn't rendering properly as a list, make sure its formatted properly

File: _commands/tools/wget.md [hunk 3]
```diff
@@ -3,22 +3,48 @@
...
+
+Download a file but save it locally under a different name
+
+Download a file and save it in a specific folder
 
 #### Example command
 
-##### Break it down
+~~~ bash
+$ wget ‐‐recursive ‐‐no-clobber ‐‐no-parent ‐‐exclude-directories /forums,/support http://example.com
```
Comment (AmritHariharan) [inline _commands/tools/wget.md]: what effect do `--no-clobber` and `--no-parent` have on this?

**Comment by edricguo** [comment]
I fixed all requested changes. Is it ok now?

**Comment by AmritHariharan** [comment]
You have been awarded credit for the HW + 2 extra points for submitting the initial PR early.

**Pull request closed without merging**